avocado/core/spawners/common.py

```diff
@@ -28,11 +29,14 @@ class SpawnerMixin:
 
     METHODS = []
 
-    def __init__(self, config=None):
+    def __init__(self, config=None, job_output_dir=None):
         if config is None:
             config = settings.as_dict()
         self.config = config
-        self.job_output_dir = None
+        if job_output_dir is None:
+            job_output_dir = os.path.join(tempfile.gettempdir(), 'avocado',
+                                          'job-results', 'test-results')
```

Comment:
Hi @beraldoleal, you are right, passing the `job` pointer is better and easier solution, because the spawner is created inside the job process. I will change that.